IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -484,13 +484,19 @@ - (void)webView:(WKWebView *)webView didReceiveServerRedirectForProvisionalNavig
     }
 }
 
+#if AD_BROKER
 - (void) webView:(WKWebView *)webView
 requestMediaCapturePermissionForOrigin:(WKSecurityOrigin *)origin
 initiatedByFrame:(WKFrameInfo *)frame
             type:(WKMediaCaptureType)type
  decisionHandler:(void (^)(WKPermissionDecision decision))decisionHandler API_AVAILABLE(ios(15.0), macos(12.0))
 {
-    if (MSID_SUPPRESS_CAMERA_CONSENT_PROMPT_IN_WEBVIEW && type == WKMediaCaptureTypeCamera)
+    
+    NSUserDefaults *userDefaults = [[NSUserDefaults alloc] initWithSuiteName:@"group.com.microsoft.azureauthenticator.sso"];
+    NSString *suppressConsentKey = @"Microsoft.Broker.Feature.sdm_suppress_camera_consent";
+    NSNumber *flagValue = [userDefaults objectForKey:suppressConsentKey];
```

Comment:
The [objectForKey](https://developer.apple.com/documentation/foundation/nsuserdefaults/1410095-objectforkey) returns an `id` instead of `NSNumber`, can we make sure the returned object conforms to `boolValue` before calling it below?

Reply:
Thanks for pointing it out, I forgot to update this section after my testing. Updated